examples/compiled/interactive_layered_crossfilter_discrete.vg.json

```diff
@@ -183,6 +182,7 @@
   ],
   "signals": [
     {"name": "childWidth", "value": 200},
+    {"name": "childHeight", "value": 200},
```

Comment:
Why does this become childHeight?

Reply:
Do you see an issue in the normalized (concat) spec? The height here is just the default, no?